File: CppCompiler/forloop16.cpp
#include<iostream>
using namespace std;

int main()
{
double i,j,n,sum=0,fact=1;
cout<<"enter no:-";
cin>>n;
for(i=1;i<=n;i++)
{
cout<<i<<"/"<<i<<"!";
if(i<n)
{
cout<<"+";
}
for(j=1;j<=i;j++)
{
fact=fact*j;
}
sum=sum+(i/fact);
fact=1;
}
cout<<"="<<sum;
return 0;
}

File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop16.java
package sample;
import java.util.*;
 class Forloop16
 {
 public static void main (String[] ar)
 {
   int i,n;
   double sum=0,z;
   System.out.println("enter range");
   Scanner m=new Scanner(System.in);
   n=m.nextInt();
   for(i=1;i<=n;i++)
  {
    System.out.print("1/"+i+"^"+i);
    if(i<n)
    {
      System.out.print(" + ");
    }
    z=Math.pow(i,i);
    sum=sum+(double)1/(i*i);
  }
  System.out.print("\nsum of seriese ="+sum);
 }
}


